CMakeLists_files.cmake

```diff
@@ -50,6 +50,7 @@ list (APPEND PUBLIC_HEADER_FILES
         opm/flowdiagnostics/ConnectionValues.hpp
         opm/flowdiagnostics/ConnectivityGraph.hpp
         opm/flowdiagnostics/Toolbox.hpp
-        opm/flowdiagnostics/utility/AssembledConnections.hpp
-        opm/flowdiagnostics/utility/RandomVector.hpp
+        opm/utility/graph/tarjan.h
```

Comment:
> While low-level, it is well documented, and it can be very useful.

Then we'll make it public when we actually need it.  In this case I prefer to be restrictive by default.